unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/Services/SceneControllerService/Tests/SceneControllerServiceShould.cs

```diff
@@ -241,6 +246,136 @@ await rpcClient.SendCrdt(new CRDTSceneMessage()
             });
         }
 
+        [UnityTest]
+        public IEnumerator CallGetCurrentStateWithoutStoredState()
+        {
+            yield return UniTask.ToCoroutine(async () =>
+            {
+                try
+                {
+                    const int TEST_SCENE_NUMBER = 666;
+                    const int ENTITY_ID = 1;
+                    const int COMPONENT_ID = 1;
+                    byte[] outgoingCrdt = new byte[] { 0, 0, 0, 0 };
+
+                    ClientRpcSceneControllerService rpcClient = await CreateRpcClient(testClientTransport);
+                    await rpcClient.LoadScene(CreateLoadSceneMessage(TEST_SCENE_NUMBER));
+
+                    bool sceneHasStateStored = true;
+                    bool getCurrentStateFinished = false;
+                    ByteString responsePayload = null;
+
+                    rpcClient.GetCurrentState(new GetCurrentStateMessage() { })
+                             .ContinueWith(crdtState =>
+                              {
+                                  sceneHasStateStored = crdtState.HasOwnEntities;
+                                  responsePayload = crdtState.Payload;
+                                  getCurrentStateFinished = true;
+                              });
+
+                    var protocol = new CRDTProtocol() { };
+                    protocol.ProcessMessage(protocol.Create(ENTITY_ID, COMPONENT_ID, outgoingCrdt));
+                    context.crdt.scenesOutgoingCrdts.Add(TEST_SCENE_NUMBER, new CRDTProtocol() { });
+                    await new WaitUntil(() => getCurrentStateFinished, 1);
+
+                    Assert.IsTrue(getCurrentStateFinished);
+                    Assert.IsFalse(sceneHasStateStored);
+
+                    using (var iterator = CRDTDeserializer.DeserializeBatch(responsePayload.Memory))
+                    {
+                        while (iterator.MoveNext())
+                        {
+                            var responseCrdt = (CRDTMessage)iterator.Current;
+                            Assert.AreEqual(responseCrdt.key1, ENTITY_ID);
+                            Assert.AreEqual(responseCrdt.key2, COMPONENT_ID);
+                            Assert.IsTrue(AreEqual(outgoingCrdt, (byte[])responseCrdt.data));
+                        }
+                    }
+                }
+                catch (Exception e)
+                {
+                    Debug.LogError(e);
```

Comment:
Why do you need to catch an exception and log it afterward? If Errors are not ignored they will make a test fail anyway